uwblueprint/sistering
# Update status when removing volunteer + use API data for table

## Ticket link

Closes #250 



## Implementation description
* Implemented the API call for the remove volunteer modal
* Connected table data with results from GraphQL API call
* Fixed minor bugs in the volunteer table, such as duplicate component keys.



## Steps to test
1. Navigate to `http://localhost:3000/admin/schedule/posting/{id}` 
2. Click the "Review" button to go to the review table
3. Check that the table is being filled with the correct data.
4. Try removing a signup.
5. Check that the signup has been removed from the table
6. Refresh the page and go to the side panel. The signup should now be uncheckmarked.



## What should reviewers focus on?
* Correct implementation & functionality


## Checklist
